### ISSUE TITLE
[Insights] createEntry date incorrect when using date option for historic entries

### ISSUE THREAD

**Issue by gruijter** [athombv/homey#716]
**What is your Homey version (Settings → System)?**

0.8.39

**What did you try to achieve?**

Create a log where the log data is from the past (could be minutes, hours, days or even months ago).

_Please include detailed steps how we can reproduce the issue_

```
logDate = new Date(dateOfLog);
Homey.log(logDate);       // result: Wed Sep 23 2015 00:00:00 GMT+0200 (CEST)
Homey.manager('insights').createEntry( logId, logValue, logDate, function(err, success){}
```

**What was the result?**

Insights is showing the log entries with the right value, but not with the right date/time. (Note, the horizontal time axis is displayed in ms. See attached example)

The exported data shows (download from insights):

```
Sat Jul 02 2016 13:34:25 GMT+0200 (CEST),-15

Sat Jul 02 2016 13:34:26 GMT+0200 (CEST),-9

Sat Jul 02 2016 13:34:26 GMT+0200 (CEST),-10

Sat Jul 02 2016 13:34:27 GMT+0200 (CEST),-10
```

**What did you expect as result?**

The entries to be displayed with the correct dates. the correct dates should be:
23/11/2015  (or in full: Wed Nov 23 2015 00:00:00 GMT+0100 (CET) )
11/3/2016
29/6/2016
29/6/2016

**Any other remarks**
The date/time that is shown when downloading from insights is the date/time when the log was created.

I have tried this before in firmware version 0.8.32 with similar results.
...
![2016-07-02](https://cloud.githubusercontent.com/assets/19174120/16540229/1b7c513c-405d-11e6-8e7b-05d0af0b0279.png)

**Comment by gruijter** [comment]
Any news on this? I still have the same issue on fw 0.10.7. When creating a log entry using a date in the past, the present date/time is logged in stead.

**Comment by WeeJeWel** [comment]
I'll look into this ASAP :)

**Comment by WeeJeWel** [comment]
Fixed!

**Comment by gruijter** [comment]
Confirmed fixed in 0.10.9. Thx!
